src/checkBoxReferenceSetSelector.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com/mendix/widget/custom/checkboxreferencesetselector/CheckBoxReferenceSetSelector" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Checkbox referenceSet Selector</name>
+    <description>Our checkBoxreferenceSetSelector widget</description>
+    <icon/>
+    <properties>
+        <property key="captionName" type="string">
+            <caption>checkBoxreferenceSetSelector</caption>
+            <category>behaviour</category>
+            <description/>
+        </property>
+        <property key="dataSourceType" type="enumeration" defaultValue="xpath">
+			<caption>Data source type</caption>
+			<category>Data source</category>
+			<description>The method for retrieving the objects.</description>
+			<enumerationValues>
+				<enumerationValue key="xpath">XPath</enumerationValue>
+				<enumerationValue key="mf">Microflow</enumerationValue>
```

Comment:
dont use acronyms unless you everybody know what it means... computers dont get slow because you use long variable names. We write code for readability. => `microflow`